sql/2020/07_SEO/json-ld-scripts-raw.sql

```diff
@@ -0,0 +1,98 @@
+# how many lson-ld scripts on a page
```

Comment:
This will eventually just be one query per file, right?

Reply:
Done. Those scripts were more of a learning curve for me. Not certain they will be in the final cut.

Reply:
Gone now, I've worked out how to do it via almanac.js